tools/unix/generate_localizations.sh

```diff
@@ -66,4 +66,8 @@ cat "$STRINGS_PATH"/{strings,types_strings}.txt> "$MERGED_FILE"
 "$TWINE" generate-all-localization-files --format jquery "$OMIM_PATH/data/countries_names.txt" "$OMIM_PATH/data/countries-strings/"
 "$TWINE" generate-all-localization-files --format jquery "$OMIM_PATH/data/strings/sound.txt" "$OMIM_PATH/data/sound-strings/"
 
+# Generate list of languages and add list in gradle.properties to be used in build.gradle in resConfig
+RES_CONFIG=$(sed -nE "s/ +([a-zA-Z]{2}(-[a-zA-Z]{2,})?) = .*$/'\1',/p" data/strings/strings.txt | sort -u | tr '\n' ' ' | sed -E 's/, $//')
+echo listLanguages="$RES_CONFIG" >> "$OMIM_PATH/android/gradle.properties"
```

Comment:
It will add the line _each time_ the script is run. The robust implementation should _add or replace existing_ line if it's already there.